task08/task08/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task08
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+           Console.WriteLine("Введите значение х");
+           double x = double.Parse(Console.ReadLine());
+           Console.WriteLine(Calc(x));
+        }
+        static double Calc(double x)
+        {
+            if (x > 1)
+            {
+                return 2 - x;
+            }
+            else if (x >= -1 & x <= 1)
+            {
+                return x * x;
+            }
+            else
+            {
+                return 1;
+            }
```

Comment:
1. Если после if следует только один оператор, то `{ }` можно не писать.
2. Если выполнение операторов внутри if заканчивается оператором return, то дальнейший else излишен.
3. Если изменить порядок проверки, то можно избежать двойного неравенства.
```suggestion
            if (x > 1)
                return 2 - x;

            if (x < -1)
                return 1;

            return x * x;
```